cscareerhub/megabot
# [misc] Add coverage/ to .gitignore

### Purpose of pull request

**Related Issue**: #

Running `yarn coverage` results in a `coverage/` directory that should be ignored by Git. 

### Pull request checklist


### Testing instructions

Run `yarn coverage` and verify that the coverage/ directory is ignored correctly.
